build/generate-bundle-name.js

```diff
@@ -0,0 +1,6 @@
+module.exports = (format, isProd) => {
+  if (format === 'cjs') {
+    return 'trae';
+  }
+  return isProd ? 'trae.umd.min' : 'trae.umd';
+};
```

Comment:
I would change this function to do a consistent usage of the `if` statement and the ternary operator. Maybe something like this:
```js
module.exports = (format, isProd) => format === 'cjs'
  ? 'trae'
  : isProd ? 'trae.umd.min' : 'trae.umd';
```

But is just a matter of preference.